core/src/main/kotlin/com/malinskiy/marathon/execution/TestBatchResults.kt

```diff
@@ -6,5 +6,14 @@ data class TestBatchResults(
     val device: Device,
     val finished: Collection<TestResult>,
     val failed: Collection<TestResult>,
-    val uncompleted: Collection<TestResult>
-)
+    val uncompleted: Collection<TestResult>,
+    val runCompletionReason: RunCompletionReason
+) {
+
+    enum class RunCompletionReason {
+        RUN_FAILED,
+        RUN_STOPPED,
```

Comment:
Stopped in our case also includes timeout

Reply:
As I understand I could delete `RUN_STOPPED_FAIL_FAST` and should use `RUN_STOPPED`

Reply:
Not really, just an FYI that run stopped can happen if the run was actually stopped and at the same time because it has timed out

Reply:
Up to you if you want to create another completion reason for timeout